src/pages/home/report/ReportView.js

```diff
@@ -1,31 +1,45 @@
 import React from 'react';
 import {Keyboard, View} from 'react-native';
 import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
 import ReportActionsView from './ReportActionsView';
 import ReportActionCompose from './ReportActionCompose';
 import {addAction} from '../../../libs/actions/Report';
 import KeyboardSpacer from '../../../components/KeyboardSpacer';
 import styles from '../../../styles/styles';
 import SwipeableView from '../../../components/SwipeableView';
+import ONYXKEYS from '../../../ONYXKEYS';
 
 const propTypes = {
     /* The ID of the report the selected report */
     reportID: PropTypes.number.isRequired,
+
+    /* Onyx Keys */
+    // Whether or not to show the Compose Input
+    session: PropTypes.shape({
+        shouldShowComposeInput: PropTypes.bool.isRequired,
```

Comment:
When first logging into an account on this branch I'm getting this error:

![image](https://user-images.githubusercontent.com/47436092/117717007-1a9f0180-b18f-11eb-849a-30c000bb82ac.png)

And the `ReportActionCompose` is not showing because, I guess because the Onyx value is missing. Then when I refresh the page everything goes back to normal.

Reply:
I _think_ this might mean we need to run an Onyx migration, but I'm not sure. Maybe we can give make this prop optional and give it a default value of true?